packages/frontend/src/components/project/ArchivedBar.tsx

```diff
@@ -0,0 +1,27 @@
+import cx from 'classnames'
+import React from 'react'
+
+import { ArchivedIcon } from '../icons/symbols/ArchivedIcon'
+
+export function ArchivedBar() {
+  return (
+    <div
+      className={cx(
+        'mb-6 flex justify-center py-2.5 md:px-0',
+        'rounded-lg bg-gray-200 dark:bg-gray-800',
+      )}
+    >
+      <span className="flex flex-row">
+        <span className="flex items-center">
+          <ArchivedIcon />
+        </span>
+        <span className="ml-2 text-base font-medium dark:text-white">
```

Comment:
```suggestion
        <span className="ml-2 text-base font-medium">
```
Is it necessary? Text defaults to white on dark mode